modules/srvrlss-using-cluster-logging-find-knative-serving-logs.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// cluster-logging-serverless.adoc
+
+[id=using-cluster-logging-to-find-Knative-Serving-logs_{context}"]
+= Using Cluster Logging to find Knative Serving logs
+
+Knative Serving provides default logs for your application. 
```

Comment:
This talks about the knative serving components specifically and has nothing to do with applications.